grails-app/migrations/0.8.x/changelog-2020-02-27-2051-insert-currency-uom-class.xml

```diff
@@ -50,4 +50,11 @@
             <where>id='CURRENCY'</where>
         </update>
     </changeSet>
+    <changeSet author="jmiranda" id="1582858163230-5">
```

Comment:
Won't this be problematic if you try to apply this changelog on db which already has it applied?

Reply:
It's only problematic if I had edited an existing changeset. You can add changesets with unique IDs anywhere in any of the changelogs and they'll run if they have never been executed.